README.md

```diff
@@ -6,18 +6,14 @@ https://bitcoincore.org
 For an immediately usable, binary version of the Bitcoin Core software, see
 https://bitcoincore.org/en/download/.
 
-Further information about Bitcoin Core is available in the [doc folder](/doc).
-
-What is Bitcoin?
-----------------
+What is Bitcoin Core?
+---------------------
 
-Bitcoin is an experimental digital currency that enables instant payments to
-anyone, anywhere in the world. Bitcoin uses peer-to-peer technology to operate
-with no central authority: managing transactions and issuing money are carried
-out collectively by the network. Bitcoin Core is the name of open source
-software which enables the use of this currency.
+Bitcoin Core connects to the Bitcoin peer-to-peer network to download and fully
```

Comment:
Nitpick: Maybe it's obvious, but shouldn't the question "What is X?" answered by a sentence that starts with "X is ..." before answering what X does?

```suggestion
Bitcoin Core is a software that connects to the Bitcoin peer-to-peer network to download and fully
```

Reply:
Sure, though given the context, I mean it's github, there's only software on here.

Reply:
It is explained in the previous sentence: "the Bitcoin Core software", so I am going to reject the change for now.